pa1k4 hide 6 and 7 (#1352)
pcdshub/pcdsdevices: docs/source/upcoming_release_notes/1352-hide-pa1k4-targets.rst

1352 hide-pa1k4-targets
#################

API Breaks
----------
- N/A

Library Features
----------------
- N/A

Device Features
---------------

> - change target from 7 to 5 and hide 6 and 7

Reviewer: ```suggestion
- For ``PA1K4`` (`CalibrationAxis`), change target count from 7 to 5 to hide targets 6 and 7
```

Author: Just update it.

Reviewer: Why did you do it manually, now I have to suggest it again

> -  for "PA1K4"('CalibrationAxis'),change target count from 7 to 5 to  hide 6 and 7

Reviewer: ```suggestion
- For ``PA1K4`` (`CalibrationAxis`), change target count from 7 to 5 to hide targets 6 and 7
```

Reviewer: I'm just going to force this through the way I suggested because I have pain later when making release notes if the notes aren't right

New Devices
-----------
- N/A

Bugfixes
--------
- N/A

Maintenance
-----------
- N/A

Contributors
------------

> - tongju

Reviewer: Github username
```suggestion
- tongju12
```
